CMakeLists.txt

```diff
@@ -14,15 +14,21 @@
 # along with roboptim-core.  If not, see <http://www.gnu.org/licenses/>.
 
 # Requires at least CMake 2.8 to configure the package.
-CMAKE_MINIMUM_REQUIRED(VERSION 2.8)
+CMAKE_MINIMUM_REQUIRED(VERSION 3.16)
 
-INCLUDE(cmake/base.cmake)
-INCLUDE(cmake/boost.cmake)
-INCLUDE(cmake/cpack.cmake)
+SET(CMAKE_CXX_STANDARD 11)
```

Comment:
This will break things on systems where the default compiler will default to C++ >= 14 (ie. all of them nowadays). Please use https://github.com/jrl-umi3218/jrl-cmakemodules/blob/master/cxx-standard.cmake instead.

Reply:
Fixed by 0ba76bb